opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -150,6 +155,19 @@ const RootComponent: FunctionComponent<RootComponentProps> = ({ queryRef }) => {
   // TODO : Use the hook useHelper when all project is pure function //
   const bannerSettings = computeBannerSettings(settings);
   const platformModuleHelpers = platformModuleHelper(settings);
+
+  useEffect(() => {
+    setAnalyticsInstance(Analytics({
+      app: 'opencti',
+      plugins: [
+        googleAnalytics({
+          measurementIds: [queryData.settings.analytics_google_analytics_v4],
+          enabled: !!queryData.settings.analytics_google_analytics_v4,
```

Comment:
At first we have only one plugin : google analytics, this is optin if no measurementId is supplied the plugin is inactive